examples/iota_lens/analysis_iotalens.py

```diff
@@ -4,7 +4,6 @@
 # Authors: Axel Huebl, Chad Mitchell
 # License: BSD-3-Clause-LBNL
 #
-
 import glob
```

Comment:
Undo formatting change:
```suggestion

import glob
```